beacon-chain/sync/initial-sync/blocks_fetcher.go

```diff
@@ -575,8 +576,9 @@ func populateBlockWithColumns(bw blocks2.BlockWithROBlobs, columns []blocks.RODa
 		return bw, errDidntPopulate
 	}
 	colsPersub := params.BeaconConfig().NumberOfColumns / params.BeaconConfig().DataColumnSidecarSubnetCount
-	if len(columns) != int(params.BeaconConfig().CustodyRequirement*colsPersub) {
-		return bw, errors.Errorf("unequal custodied columns provided, got %d instead of %d", len(columns), int(params.BeaconConfig().CustodyRequirement))
+	subnetCount := peerdas.CustodySubnetCount()
```

Comment:
@nisdas could you please checks changes in this file are OK?

Reply:
Looks good to me